spec/lib/rev/get_attachment_content_spec.rb

```diff
@@ -62,12 +62,11 @@
     action.must_raise Rev::NotFoundError
   end
 
-  # requesting conversion from pdf to :docx from order TC0263917003
```

Comment:
Optional: If you look at line 16 if you can see we wrap a couple tests in a 'success' block which groups the tests which makes it a little more organized. The tests that test failure are currently not wrapped in a block like that. Consider wrapping these two tests as a way to clean up this code a little.

`describe 'not allowed' do`
tests
`end`

Reply:
![screen shot 2018-07-10 at 12 16 23 pm](https://user-images.githubusercontent.com/6540994/42526323-1a7e816e-843b-11e8-9ef7-cedc3e95eaf4.png)

Makes the tests organized when running rake test